content/en/docs/plugin-guide/plugin-policy-engine.md

```diff
@@ -39,7 +41,10 @@ In addition to the plugin, you need access to an Open Policy Agent (OPA) deploym
 {{< tabs name="enable-plugin" >}}
 {{% tab name="Operator" %}}
 
-This is a sample configuration to use with the Spinnaker operator:
+You can use the sample configuration to install the plugin, but keep the following in mind:
+
+- The `patchesStrategicMerge` section for each service is unique to that service. Do not reuse the snippet from one service for the other services.
```

Comment:
```suggestion
- The `patchesStrategicMerge` section for each service is unique. Do not reuse the snippet from one service for the other services.
```

Sorry I didn't catch this in my earlier suggestion. Just an attempt to remove repetition.